main.js

```diff
@@ -59,7 +60,20 @@ function recursiveDescend (node, scene, gui) {
 };
 
 mirror.doc.on('changes', debounce(rebuild, 1000));
-rebuild();
+
+// Would use `URLSearchParams`, but it's supported by only Firefox atm.
+var qsUrl = /[\?&]url=(.*)/i.exec(window.location.search);
+if (qsUrl && qsUrl[1]) {
```

Comment:
rather than running a regex match here, and again in load.js, we should always just pass the string to loadScene and do the regex matching once.